components.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "https://ui.shadcn.com/schema.json",
+  "$schema": "https://ui.shadcn.com/schema.j`son",
```

Comment:
It looks like a typo with ` (Grave accent)

Reply:
Sure! My mistake. Fixed.